### PR TITLE
Revised core player cards as duplicates

### PR DESCRIPTION
**Pull request by zzorba**
Does not include investigators/signatures, since that might break deckbuilding.
Only card I am uncertain about is the order of Charisma/Relic Hunter (could not see number in leaks)

### CODE REVIEW

File: packs.json [hunk 1]
```diff
@@ -8,6 +8,15 @@
...
         "size": 184
     },
+    {
+        "cgdb_id": 1,
+        "code": "rcore",
+        "cycle_code": "core",
+        "date_release": "2020-10-01",
+        "name": "Revised Core Set",
+        "position": 2,
+        "size": 184
```
Comment (drawntotheflame) [inline packs.json:18]: Do you want to increase this size as we have 195 cards, above?

File: pack/core/revised_core.json [hunk 2]
```diff
@@ -0,0 +1,615 @@
...
+    {
+        "code": "01693",
+        "duplicate_of": "02194",
+        "pack_code": "rcore",
+        "position": 193
+    },
+    {
+        "code": "01694",
+        "duplicate_of": "02158",
+        "name": "Charisma",
```
Comment (drawntotheflame) [inline pack/core/revised_core.json:604]: Do you need these two names here? As they are duplicates?

**Review by drawntotheflame — COMMENTED** [review]
Good job @zzorba

**Comment by zzorba** [comment]
@Kamalisk I think this is ready to be merged now

**Pull request merged**